Day-II-Calculator/calculator/src/App.js

```diff
@@ -1,21 +1,35 @@
 import React from 'react';
 import './App.css';
+import Button from './components/ButtonComponents/NumberButton'
+
+const buttons = [
+  {
+  zero: '0',  
+  one: '1',
+  two: '2', 
+  three: '3',
+  four: '4',
+  five: '5',
+  six: '6',
+  seven: '7',
+  eight: '8',
+  nine: '9',
+  divide: '/',
+  multiply: 'x',
+  subtract: '-',
+  add: '+',
+  equal: '='
+  }
+]
 
 const App = () => {
   return (
-    <div>
-      <h3>Welcome to React Calculator</h3>
-      <p>
-        We have given you a starter project. You'll want to build out your
-        components in their respective files, remove this code and replace it
-        with the proper components.
-      </p>
-      <p>
-        <strong>
-          Don't forget to `default export` your components and import them here
-          inside of this file in order to make them work.
-        </strong>
-      </p>
+    <div className = 'calculator'>
```

Comment:
It's a unique approach of doing this project, but it was suppose to help you grasp the concept of re-using one component for multiple things.